docs/DataProviderWriting.md

```diff
@@ -687,7 +687,7 @@ Here is an example implementation, that you can use as a base for your own Data
 
 ```js
 import { fetchUtils } from 'react-admin';
-import { stringify } from 'query-string';
```

Comment:
Do we need to do that? I see no upgrade guide in query-string for that change.

Reply:
yeah, this package just export default now